src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/calculator';
 
 function App() {
```

Comment:
* Kindly use class based components as stated in the project [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Use%20class%20based%20components.). as in future milestone you would be  asked to refactor all components to functional components